unityparser/parser.py

```diff
@@ -5,6 +5,9 @@
 
 class Parser(YamlParser):
 
+    # UNITY: used to store data after the anchor
+    extra_anchor_data = {}
```

Comment:
Like it's `Serializer` counterpart, it should belong to `yaml.composer.Composer` _instance_ attributes.
In this case though we don't want to clear the attribute for the same reason we don't clear `anchors`:
https://github.com/socialpoint-labs/unity-yaml-parser/blob/6dba73dc6ff8c261759fbb7fcd690976bb156c10/unityparser/composer.py#L4-L18